src/tools/pub/cmd/pub-add.md

```diff
@@ -39,6 +39,26 @@ $ dart pub add foo:'>2.0.0 <3.0.1'
 If `dart pub add <package>:<constraint>` is an existing dependency,
 it will update the constraint.
 
+## Adding Dependency Overrides
+
+Starting in Dart 3, `dart pub add` supports adding dependency overrides in a similar fashion to dev dependencies. To add a dependency override, use the following syntax:
+`dart pub add override:<package>:<version>`
+
+For example, to add an override for the `foo` package version `1.0.0`, run the following command:
```

Comment:
Good example. A few things here:

- We break up lines >= 80 chars long up with something called [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks)
- Consider making "For example" bold, since the example might be enough for some readers.
- I think "add an override for x package version" might be a bit confusing, as a dependency override doesn't override the specified package version, it specifies that version as the override.

How does this sound?

```suggestion
**For example**, to override all references to `package:foo`
to use version `1.0.0` of the package,
run the following command:
```